fa25-team-b/scripts/age_analysis.py

```
@@ -0,0 +1,378 @@
+
+    # Create age bins
+    valid_mask = df['Age'].notna()
+    valid_ages = df.loc[valid_mask, 'Age'].astype(float)
+
+    min_age = 18.0
+    max_age = float(valid_ages.max()) if len(valid_ages) else 78.0
+    if max_age <= min_age:
+        max_age = 78.0
+
+    bins = np.linspace(min_age, max_age, 7)
+    labels = [f"{int(bins[i])}–{int(bins[i+1])}" for i in range(len(bins) - 1)]
```

Ruff (0.14.8)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)